yt_dlp/downloader/iqaudio.py

```diff
@@ -0,0 +1,44 @@
+from .fragment import FragmentFD
+import gzip
+
+
+class IqAudioFragmentFD(FragmentFD):
+    """
+    Downloads a list of audio fragments, the first fragment is always a gzipped header.
+    """
+
+    FD_NAME = 'IqAudioFragmentDownloader'
+
+    def real_download(self, filename, info_dict):
+        format = None
+        for f in info_dict['formats']:
+            if f['format_id'] == info_dict['format_id']:
+                format = f
+                break
+
+        if format is None:
+            return
+
```

Comment:
Remove these lines and rename `format` to `info_dict`
`info_dict` here is the `format`

Reply:
Aahh I see, I hadn't realized!